sfge/src/test/java/com/salesforce/graph/source/supplier/GlobalMethodSupplierTest.java

```diff
@@ -0,0 +1,46 @@
+package com.salesforce.graph.source.supplier;
+
+import com.salesforce.collections.CollectionUtil;
+import org.junit.jupiter.api.Test;
+
+public class GlobalMethodSupplierTest extends BaseSourceSupplierTest {
+
+    /** Non-test global-scoped methods should be returned by this supplier. */
+    @Test
+    public void supplierLoadsGlobalMethods() {
```

Comment:
Equivalent to `PathEntryPointUtilTest#getPathEntryPoints_includesGlobalMethods` and `PathEntryPointUtilTest#testGetGlobalmethods`